adafruit_bh1750.py

```diff
@@ -182,31 +189,31 @@ class BH1750:  # pylint:disable=too-many-instance-attributes
     mode = RWBitfields(2, 4)
     resolution = RWBitfields(2, 0)
 
-    def __init__(self, i2c, address=_BH1750_DEFAULT_ADDRESS):
+    def __init__(self, i2c: I2C, address: int = _BH1750_DEFAULT_ADDRESS) -> None:
 
         self.i2c_device = i2c_device.I2CDevice(i2c, address)
         self._buffer = bytearray(2)
         self._settings_byte = 0
 
         self.initialize()
 
-    def initialize(self):
+    def initialize(self) -> None:
         """Configure the sensors with the default settings."""
         self.mode = Mode.CONTINUOUS  # pylint:disable=no-member
         self.resolution = Resolution.HIGH  # pylint:disable=no-member
 
     @property
-    def _settings(self):
+    def _settings(self) -> int:
         return self._settings_byte
 
     @_settings.setter
-    def _settings(self, value):
+    def _settings(self, value: int) -> None:
         self._settings_byte = value
         self._write(self._settings_byte)
         sleep(0.180)  # worse case time to take a new measurement
 
     @property
-    def _raw_reading(self):
+    def _raw_reading(self) -> bytearray:
```

Comment:
Based on the [format used in `unpack_from()`](https://docs.python.org/3/library/struct.html#format-characters), the return type annotation is `int`.